src/staff.ts

```diff
@@ -10,11 +10,12 @@ import * as db from './db';
  * @return {String} text
  */
 function ticketMsg(name, message) {
-  return `${cache.config.language.dear} `+
-    `${middleware.escapeText(name)},\n\n`+
-    `${middleware.escapeText(message.text)}\n\n`+
-    `${cache.config.language.regards}\n`+
-    `${message.from.first_name}`;
+  const first_name = !cache.config.mod_anonymous ? `${message.from.first_name}` : "";
```

Comment:
What I initially meant is that you remove the new line only if the first name is not sent. So when you `from.first_name` ist empty it does not add the new line at the end.
So please add something like this:

    const first_name = !cache.config.mod_anonymous ? `${message.from.first_name}\n` : "";